pkg/interface/src/views/components/VirtualScroller.tsx

```diff
@@ -309,15 +302,15 @@ export default class VirtualScroller extends Component<VirtualScrollerProps, Vir
             height: element.offsetHeight,
             element
           });
-          _.debounce(this.recalculateTotalHeight, 500)();
+          this.recalculateTotalHeight();
```

Comment:
Oh shoot, this is where the debouncer was getting duplicated, huh?

Reply:
Yeah, perf gains are actually pretty nice